_includes/one-click-payments.md

```diff
@@ -162,7 +164,7 @@ Content-Type: application/json
         "payeeId": "{{ page.merchantId }}",
         "isDeleted": true,
         "isPayeeToken": false,
-        "cardBrand": "Visa|MasterCard|...",
+        "cardBrand": "Visa | MasterCard | ...",
```

Comment:
No pipes in examples, please. Explain the possible values in below property tables instead.

```suggestion
        "cardBrand": "Visa",
```